### PR TITLE
Created 18-792 Guide

### PR DESCRIPTION
**Pull request by danmohd**
# Summary of Changes

- Please indicate the classes you are modifying:
  - 18-792

# Checklist:

Make sure that your contributions:

- [x] Are accurate and will not mislead the reader
- [x] Do not encourage or cause academic integrity violations
- [x] Do not offend professors or unfairly criticize the university

### CODE REVIEW

File: electives/18792.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- **Spectral Subtraction and Phase Voboding**
+- **Random/Stochastic Processes**
+- **Power Spectral Density Estimation**
+- **Linear Predictive Coding**
+- **Lattice Filters**
+- **Adaptive Filtering**
+- **Array Processing and Adaptive Array Processing**
+- **Miscellaneous Topics: Homomorphic Signal Processing, Wavelet Representations, Robust Automatic Speech Recognition, etc.**
+
+## Course Logistics
```
Comment (mikinty) [inline electives/18792.md:25]: Markdown header style 

```suggestion
## Course Logistics

```

File: electives/18792.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# 18-792 : Advanced Digital Signal Processing
+
+| Category | Difficulty |
+|:-:       | :-:        |
+| HW       | 3          |
+| Exams    | 4          |
+
+This is a course on Advanced Digital Signal Processing and should typically be taken after having done a course on Signals and Systems (18-290) and/or Digital Signal Processing (18-491/18-691) as well as a course on Probability Theory. The course go fairly deep into the topics mentioned below as well as their applications. Homework problems will ask you to implement a lot of these algorithms, usually on speech or music data, and give you a taste of the applications of the topics covered. The signals examined are usually one dimensional signals.
+
+## Topics Covered
```
Comment (mikinty) [inline electives/18792.md:10]: Markdown header style 

```suggestion
## Topics Covered

```

File: electives/18792.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# 18-792 : Advanced Digital Signal Processing
+
+| Category | Difficulty |
+|:-:       | :-:        |
+| HW       | 3          |
+| Exams    | 4          |
+
+This is a course on Advanced Digital Signal Processing and should typically be taken after having done a course on Signals and Systems (18-290) and/or Digital Signal Processing (18-491/18-691) as well as a course on Probability Theory. The course go fairly deep into the topics mentioned below as well as their applications. Homework problems will ask you to implement a lot of these algorithms, usually on speech or music data, and give you a taste of the applications of the topics covered. The signals examined are usually one dimensional signals.
```
Comment (mikinty) [inline electives/18792.md]: Do you mind adding a link to 18-290 since it has a guide in this repo?

**Review by mikinty — APPROVED** [review]
Looks good, thank you!

File: electives/18792.md (hunk 3 above)
Comment (danmohd) [reply electives/18792.md]: Done in ca1c911c146a17b80ab8a61a67aca297b09bea73!

**Comment by mikinty** [comment]
Feel free to merge whenever you're ready

**Comment by danmohd** [comment]
Sure. Won't I need write access for that?

**Comment by mikinty** [comment]
> Sure. Won't I need write access for that?

Do you not have the permissions to merge this pull request?

image

**Comment by danmohd** [comment]
Nope.

**Comment by mikinty** [comment]
Noted, thanks for your work!

**Pull request merged**